backend/molgenis-emx2/src/main/java/org/molgenis/emx2/Column.java

```diff
@@ -390,6 +390,33 @@ public boolean isReference() {
     return getColumnType().isReference();
   }
 
+  public boolean isReferenceWithRefback() {
+    if (this.isReference()) {
+      for (Column c : this.getRefTable().getColumns()) {
+        if (c.isRefback()
+            && c.getRefTableName().equals(this.getTableName())
```

Comment:
added check for schema name also

Reply:
My tests fail because of this. getSchema().getName() of the reftable seems te be null. Am I initialising the tables incorrectly in the tests?

Reply:
hmm, I need to take quick look. Schema is only available when using data retrieved from database. But might be something else happening here.